src/server/handler/userHandler.ts

```diff
@@ -2,41 +2,31 @@ import * as express from "express";
 import Handler from "./handler";
 import User from "../types/user";
 import * as utils from "../utils";
+import {Config} from "../../config";
 
 export default class UserHandler extends Handler {
 
     constructor() {
         super();
-        this.getRouter().post("/signUp", this.signUp.bind(this));
-        this.getRouter().post("/signIn", this.signIn.bind(this));
+        this.getRouter().get("/user", this.getUser.bind(this));
+        this.getRouter().get("/logout", this.logout.bind(this));
     }
 
-    private async signUp(req: express.Request, res: express. Response): Promise<void> {
-        try {
-            if (await User.findOne({ email: req.body.email }))
-                throw new Error("Email is already in use");
-            req.body.password = utils.hashPassword(req.body.password);
-            const user = await User.create(req.body);
-            res.json({
-                "status": "success",
-                "token": user._id
-            });
-        } catch (e) {
-            utils.handleError(e, res);
-        }
+    private async getUser(req: express.Request, res: express.Response): Promise<void> {
+        res.json({
+            // @ts-ignore
```

Comment:
why is this here? consider using https://github.com/hopperteam/hopper-server/blob/master/src/handler/authMiddleware.ts#L10-L16